wcc-contentful-app/lib/wcc/contentful/app/middleware/publish_at.rb

```diff
@@ -31,5 +42,82 @@ def before(time)
 
       Time.zone.now <= Time.zone.parse(time)
     end
+
+    if defined?(ActiveJob::Base)
+      def maybe_drop_job(entry)
```

Comment:
`drop_job` makes it sound like we're not doing this... maybe `enqueue_job` instead?

Reply:
`maybe_enqueue_job`?  It only drops a job if the entry's `publish_at` or `unpublish_at` is in the future.